vendor/ansible/roles/contiv_network/tasks/ovs.yml

```diff
@@ -11,7 +15,7 @@
         url: "https://cisco.box.com/shared/static/zzmpe1zesdpf270k9pml40rlm4o8fs56.rpm",
```

Comment:
The local build(swarm) has these versions locally. K8s doesn't need these any more as the ovs stuff is in the containers. Defaults can be moved to use openstack kilo repositories - not sure why, but we disable the repo explicitly in the default contiv/netplugin vagrant setup. For RHEL it requires an openstack subscription.